sleap/gui/app.py

```diff
@@ -311,7 +311,8 @@ def _create_video_player(self):
         self.player = QtVideoPlayer(
             color_manager=self.color_manager, state=self.state, context=self.commands
         )
-        self.player.changedPlot.connect(self._after_plot_update)
+        self.player.changedPlot.connect(self._after_plot_change)
+        self.player.updatedPlot.connect(self._after_plot_update)
```

Comment:
I renamed the existing `_after_plot_update` function to `after_plot_change` and created a new function `_after_plot_update`